delta-lake/src/integration/java/org/apache/iceberg/delta/TestSnapshotDeltaLakeTable.java

```diff
@@ -113,6 +124,8 @@ public static Object[][] parameters() {
   @Rule public TemporaryFolder temp3 = new TemporaryFolder();
   @Rule public TemporaryFolder temp4 = new TemporaryFolder();
   @Rule public TemporaryFolder temp5 = new TemporaryFolder();
+  @Rule public TemporaryFolder temp6 = new TemporaryFolder();
+  @Rule public TemporaryFolder temp7 = new TemporaryFolder();
```

Comment:
can we try to avoid creating more temp folders? We should be able to reuse the same one for each test

Reply:
Thank you for your suggestion. I misunderstood the logic of this temp folder. I revised the code to reuse the same one for each test